docs/user-guides/pallet-proxy.md

```diff
@@ -0,0 +1,79 @@
+
+* **announce**(real, callHash) - used to record the announcement of a proxy transaction made by proxy accounts that have a delay requirement. This function emits an Announced event.
+
+* **anonymous**(proxyType, delay, index) - generates a new account with an inaccessible private key, making it inaccessible. The sender assumes the role of a proxy for this account, based on the specified type and delay. Caution: If the proxy is removed, the primary account will become inaccessible. This function emits an AnonymousCreated event.
+
+* **killAnonymous**(spawner, proxyType, index, height, extIndex) - removes a previously spawned anonymous proxy.
+
+* **proxy**(real, forceProxyType, call) - performs a transaction as a proxy. This function emits a ProxyExecuted event.
+
+* **proxyAnnounced**(delegate, real, forceProxyType, call) - executes a transaction as a proxy and removes any previous corresponding announcements. This function emits a ProxyExecuted event.
+
+* **rejectAnnouncement**(delegate, callHash) - if the sender is a prime account, it removes a specific announcement from their proxy account. 
+
+* **removeAnnouncement**(real, callHash) - if the sender is a proxy account, it removes a specific announcement to their prime account.
+
+* **removeProxies()** - unregisters all proxy accounts associated with the sender.
+
+* **removeProxy**(delegate, proxyType, delay) - unregisters a specific proxy account linked to the sender. This function emits a ProxyRemoved event.
+
+
+**Storage Methods**
+---
+The proxy pallet provides the following read-only storage methods to retrieve chain state data:
+
+* **announcements**(AccountId20) - retrieves all announcements made by the specified proxy account.
+
+* **palletVersion**() - retrieves the current version of the pallet.
+
+* **proxies**(AccountId20) - retrieves a map and count of all proxy accounts associated with a specified primary account.
+
+
+**Constants**
+---
+The proxy pallet provides the following read-only functions to retrieve pallet constants:
+
+* **announcementDepositBase()** - retrieves the base amount of currency required to reserve when creating an announcement.
+
+* **announcementDepositFactor()** - retrieves the amount of currency required per announcement made.
+
+* **maxPending()** - retrieves the maximum number of time-delayed announcements allowed to be pending.
+
+* **maxProxies()** - retrieves the maximum number of proxies permitted for a single account.
+
+* **proxyDepositBase()** - retrieves the base amount of currency required to reserve when creating a proxy.
+
+* **proxyDepositFactor()** - retrieves the amount of currency required per proxy added.
```

Comment:
If you want to cover these, I'd suggest putting info inside a table with a short explanation.